fastlane_bot/helpers/txhelpers.py

```diff
@@ -744,109 +737,70 @@ def build_tx(
-        self.ConfigObj.logger.info(
-            f"[helpers.txhelpers.submit_private_transaction] Submitted transaction to Flashbots RPC, response: {response}"
-        )
+
         if response != 400:
-            tx_hash = response.get("result")
-            try:
-                tx_receipt = self.web3.eth.wait_for_transaction_receipt(tx_hash)
-                tx_hash = tx_receipt["transactionHash"]
-                return tx_hash
-            except TimeExhausted as e:
-                self.ConfigObj.logger.info(
-                    f"[helpers.txhelpers.submit_private_transaction] Transaction stuck in mempool for 120 seconds, cancelling."
-                )
-                self.cancel_private_transaction(signed_arb_tx, block_number)
-                return None
-        else:
             self.ConfigObj.logger.info(
-                f"[helpers.txhelpers.submit_private_transaction] Failed to submit transaction to Flashbots RPC"
+                f"[helpers.txhelpers.submit_private_transaction] Submitted transaction to Flashbots succeeded"
             )
-            return None
-
-    def cancel_private_transaction(self, arb_tx, block_number: int):
-        self.ConfigObj.logger.info(
-            f"[helpers.txhelpers.cancel_private_transaction] Attempting to cancel tx to Flashbots, please hold."
-        )
-        arb_tx["data"] = ""
-        signed_arb_tx = self.sign_transaction(arb_tx).rawTransaction
-        signed_tx_string = signed_arb_tx.hex()
-        max_block_number = hex(block_number + 10)
-        params = [
-            {
-                "tx": signed_tx_string,
-                "maxBlockNumber": max_block_number,
-                "preferences": {"fast": True},
-            }
-        ]
-        response = self.alchemy.core.provider.make_request(
-            method="eth_sendPrivateTransaction",
-            params=params,
-            method_name="eth_sendPrivateTransaction",
-            headers=self._get_headers,
-        )
-        if response != 400:
+            return self._submit_transaction(response.get("result"))
+        else:
             self.ConfigObj.logger.info(
-                f"[helpers.txhelpers.cancel_private_transaction] Submitted cancellation to Flashbots RPC, response: {response}"
+                f"[helpers.txhelpers.submit_private_transaction] Submitted transaction to Flashbots failed with response = {response}"
             )
             return None
-        else:
+
+    def _submit_transaction(self, tx_hash) -> str:
+        try:
+            tx_receipt = self.web3.eth.wait_for_transaction_receipt(tx_hash)
+            assert tx_hash == tx_receipt["transactionHash"]
+            return tx_hash
+        except TimeExhausted as e:
             self.ConfigObj.logger.info(
-                f"[helpers.txhelpers.cancel_private_transaction] Failed to submit cancellation to Flashbots RPC"
+                f"[helpers.txhelpers._submit_transaction] Transaction stuck in mempool for 120 seconds, cancelling."
```

Comment:
Can we change this message to:
f"[helpers.txhelpers._submit_transaction] Transaction stuck in mempool for 120 seconds, moving on."

Since we are no longer submitting a cancellation.

Reply:
Ah yeah, that's right.
We should probably also just use "timeout" instead of "120 seconds" because this is largely depends on the internal implementation of web3.py and/or the remote node in use.

Reply:
Fixed